doc/psyad/user_guide/psyad_command.rst

```diff
@@ -61,42 +61,92 @@ by the command:
 .. parsed-literal::
 		
   > psyad -h
-    usage: psyad [-h] [-v] [-oad OAD] filename
+    usage: psyad [-h] [-oad OAD] [-v] [-t] [-otest TEST_FILENAME] -a ACTIVE [ACTIVE ...] -- filename
```

Comment:
It would be nice but I don't think so. The problem is that doctest does not find and execute the psyad script, or at least I can't see how to make it do that.